library/src/main/java/com/novoda/downloadmanager/lib/DefaultNotificationInitialiser.java

```diff
@@ -0,0 +1,14 @@
+package com.novoda.downloadmanager.lib;
+
+import android.app.NotificationManager;
+import android.content.Context;
+import android.support.v4.app.NotificationCompat;
+
+import com.novoda.downloadmanager.notifications.NotificationInitialiser;
+
+class DefaultNotificationInitialiser implements NotificationInitialiser {
+    @Override
+    public NotificationCompat.Builder initNotificationBuilder(Context context, NotificationManager notificationManager) {
+        return new NotificationCompat.Builder(context);
```

Comment:
Right now it looks a bit weird to have `NotificationManager` passed in here as it's never used, but when an app that targets Android O wants to implement this interface, they will need to call `setNotificationChannel` on the `NotificationManager` as well as the `Builder` they will return, or their notifications won't work.